src/ios/CDVWKInAppBrowser.m

```diff
@@ -19,10 +19,6 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 #import "CDVWKInAppBrowser.h"
 
-#if __has_include("CDVWKProcessPoolFactory.h")
-#import "CDVWKProcessPoolFactory.h"
-#endif
```

Comment:
We're not using this, and when I looked it up, it's not really very helpful as the implementation only ever allocates a single instance, and then keeps reusing it.

https://github.com/apache/cordova-plugin-wkwebview-engine/blob/8735c4615a1135bf7d5c50753e44616641236664/src/ios/CDVWKProcessPoolFactory.m